code/game/machinery/vending_vr.dm

```diff
@@ -389,116 +389,116 @@
-					/obj/item/clothing/mask/bandana/green = 100,
-					/obj/item/clothing/mask/bandana/red = 100,
-					/obj/item/clothing/mask/surgical = 200)
+					/obj/item/clothing/accessory/sweater/flowersweater = 50,
+					/obj/item/clothing/accessory/sweater/redneck = 50,
+					/obj/item/clothing/accessory/tie = 50,
+					/obj/item/clothing/accessory/tie/horrible = 50,
+					/obj/item/clothing/accessory/tie/white = 50,
+					/obj/item/clothing/accessory/tie/navy = 50,
+					/obj/item/clothing/accessory/tie/yellow = 50,
+					/obj/item/clothing/accessory/tie/darkgreen = 50,
+					/obj/item/clothing/accessory/tie/black = 50,
+					/obj/item/clothing/accessory/tie/red_long = 50,
+					/obj/item/clothing/accessory/tie/red_clip = 50,
+					/obj/item/clothing/accessory/tie/blue_long = 50,
+					/obj/item/clothing/accessory/tie/blue_clip = 50,
+					/obj/item/clothing/accessory/tie/red = 50,
+					/obj/item/clothing/accessory/wcoat = 50,
+					/obj/item/clothing/accessory/wcoat/red = 50,
+					/obj/item/clothing/accessory/wcoat/grey = 50,
+					/obj/item/clothing/accessory/wcoat/brown = 50,
+					/obj/item/clothing/accessory/wcoat/gentleman = 50,
+					/obj/item/clothing/accessory/wcoat/swvest = 50,
+					/obj/item/clothing/accessory/wcoat/swvest/blue = 50,
+					/obj/item/clothing/accessory/wcoat/swvest/red = 50,
+					/obj/item/weapon/storage/wallet = 50,
+					/obj/item/weapon/storage/wallet/poly = 50,
+					/obj/item/weapon/storage/wallet/womens = 50,
+					/obj/item/weapon/lipstick = 50,
+					/obj/item/weapon/lipstick/purple = 50,
+					/obj/item/weapon/lipstick/jade = 50,
+					/obj/item/weapon/lipstick/black = 50,
+					/obj/item/clothing/ears/earmuffs = 50,
+					/obj/item/clothing/ears/earmuffs/headphones = 50,
+					/obj/item/clothing/ears/earring/stud = 50,
+					/obj/item/clothing/ears/earring/dangle = 50,
+					/obj/item/clothing/gloves/ring/mariner = 50,
+					/obj/item/clothing/gloves/ring/engagement = 50,
+					/obj/item/clothing/gloves/ring/seal/signet = 50,
+					/obj/item/clothing/gloves/ring/seal/mason = 50,
+					/obj/item/clothing/gloves/ring/material/plastic = 50,
+					/obj/item/clothing/gloves/ring/material/steel = 50,
+					/obj/item/clothing/gloves/ring/material/gold = 100,
+					/obj/item/clothing/glasses/eyepatch = 50,
+					/obj/item/clothing/glasses/gglasses = 50,
+					/obj/item/clothing/glasses/regular/hipster = 50,
+					/obj/item/clothing/glasses/rimless = 50,
+					/obj/item/clothing/glasses/thin = 50,
+					/obj/item/clothing/glasses/monocle = 50,
+					/obj/item/clothing/glasses/goggles = 50,
+					/obj/item/clothing/glasses/fluff/spiffygogs = 50,
+					/obj/item/clothing/glasses/fakesunglasses = 50,
+					/obj/item/clothing/glasses/fakesunglasses/aviator = 50,
+					/obj/item/clothing/mask/bandana/blue = 50,
+					/obj/item/clothing/mask/bandana/gold = 50,
+					/obj/item/clothing/mask/bandana/green = 50,
+					/obj/item/clothing/mask/bandana/red = 50,
+					/obj/item/clothing/mask/surgical = 50)
```

Comment:
This should be 100 as well

Reply:
![image](https://user-images.githubusercontent.com/24830358/84851324-3f135c00-b05a-11ea-9870-11d71d78918a.png)
Bandanas are 1 point tho.